Wire-iOS/Sources/Components/Settings+Logging.swift

```diff
@@ -45,18 +45,14 @@ extension Settings {
     
     /// Loads from user default the list of logs that are enabled
     @objc public func loadEnabledLogs() {
-        let avsTag = "AVS"
+        var tagsToEnable = Set(arrayLiteral: "AVS", "Network", "SessionManager", "Conversations", "calling", "link previews", "event-processing", "SyncStatus", "OperationStatus", "Push", "Crypto", "cryptobox")
```

Comment:
Why not use the array literal directly?

```suggestion
        var tagsToEnable: Set<String> = ["AVS", "Network", "SessionManager", "Conversations", "calling", "link previews", "event-processing", "SyncStatus", "OperationStatus", "Push", "Crypto", "cryptobox"]
```

Reply:
🤷‍♂